application/event_processor/reporting.py

```diff
@@ -57,3 +58,28 @@ def log_unmatched_nhsuk_pharmacies(nhs_entity: NHSEntity) -> None:
             "nhsuk_postcode": nhs_entity.Postcode,
         },
     )
+
```

Comment:
What your opinion to the suggestion below?

Can all functions in this file be renamed to a single standard and file name changed to `logging.py`.

Reply:
Sounds good to me. I am happy to change it but there are a few places where we are using **log** and **report** for the same purpose in a few places.

Reply:
Yes, please standardising it now will be easier than we different people write new functions some with each standard